README.md

```diff
@@ -18,9 +18,13 @@ The three programs communicate through the network. By default, they bind and co
 
 If the simulator is unavaiable, the rest of the program will work and will try to reconnect in a loop. Once the simulator is back to life, the programs will operate normally. 
 
+## Requirements
+
+The SDL2 system libs. Follow [this](https://github.com/veandco/go-sdl2#requirements) guide.

```

Comment:
Nice! But if we are gonna list requirements, why not listing them all?

```suggestion
- GNU Make (preinstalled in most UNIX systems such as Linux, Mac, Mingw-w64...)
- A suitable [Go language distribution](https://golang.org/). 
- The SDL2 system libs. Follow [this guide](https://github.com/veandco/go-sdl2#requirements).
```

Reply:
Well said !